3-Authorization-II/3-App-CallAPI/README.md

```diff
@@ -0,0 +1,393 @@
+---
+page_type: sample
+name: Enable your Java Spring Boot daemon to sign in users with Azure Active Directory and call a protected Web api
+description: Enable your Java Spring Boot daemon to sign in users in your Azure AD tenant and call a Web api protected with Spring Security and Microsoft Identity Platform
+languages:
+ - java
+products:
+ - azure-active-directory
+ - msal-java
+urlFragment: ms-identity-java-spring-tutorial
+extensions:
+- services: ms-identity
+- platform: Java
+- endpoint: AAD v2.0
+- level: 200
+- client: Java Spring Daemon
+- service: Java Spring Web API
+---
+
+# Enable your Java Spring Boot daemon to sign in users with Azure Active Directory and call a protected Web api
```

Comment:
> # Enable your Java Spring Boot daemon to sign in users with Azure Active Directory and call a protected Web api

Daemon app does not sign in users. Should probably update this in sample.json and then generate again